plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -391,13 +393,15 @@ void GSSettingsDlg::UpdateControls()
 
 	if(ComboBoxGetSelData(IDC_RENDERER, i))
 	{
-		bool dx9 = i >= 0 && i <= 2 || i == 14;
-		bool dx11 = i >= 3 && i <= 5 || i == 15;
-		bool ogl = i >= 12 && i <= 13 || i == 17;
-		bool hw = i == 0 || i == 3 || i == 12;
-		//bool sw = i == 1 || i == 4 || i == 10 || i == 13;
-		bool ocl = i >= 14 && i <= 17;
+		GSRendererType renderer = static_cast<GSRendererType>(i);
 
+		bool dx9 = renderer == GSRendererType::DX9_HW || renderer == GSRendererType::DX9_SW || renderer == GSRendererType::DX9_Null || renderer == GSRendererType::DX9_OpenCL;
+		bool dx11 = renderer == GSRendererType::DX1011_HW || renderer == GSRendererType::DX1011_SW || renderer == GSRendererType::DX1011_Null || renderer == GSRendererType::DX1011_OpenCL;
+		bool ogl = renderer == GSRendererType::OGL_HW || renderer == GSRendererType::OGL_SW || renderer == GSRendererType::OGL_OpenCL;
+		
```

Comment:
I am not exactly sure what you mean. On an empty new line there shouldn't be any whitespaces except for line feeds? Or did I miss a trailing whitespace after some code line and you marked the wrong line? At least in my local copy I can not find any trailing whitespaces at this point except for the "\n\t\t\n".

I removed now the two tabs after the inserted newline feed. I hope that is what you meant. "\n\t\t\n" -> "\n\n"

I actually considered including such tabs as better solution as it instantly shows the correct indent if you want to extent code. But I also understand the problems of trailing whitespaces.

If you meant something else I will fix it.